### PR TITLE
gorilla-bin: init at 1.5.3.7

### PR DESCRIPTION
**Pull request by namore**
###### Motivation for this change

Add password manager "gorilla" (currently available for various operating systems, e.g., Debian or FreeBSD, but not yet nix*).

###### Things done

- [X] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] Linux
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

##### Caveats

Only 64 bit Linux platform supported, since package is based on binary version.

---

### CODE REVIEW

**Comment by 7c6f434c** [comment]
Is the build process complicated? Maybe call this gorilla-bin?

**Comment by namore** [comment]
@7c6f434c, I followed your advice and postfixed the package name to gorilla-bin.

Also: yes, I consider the build process rather complex for nix (e.g., binary shared objects in the source tree for common platforms..).

**Comment by namore** [comment]
If I interpret the CI build logs correctly, gtk requires a rebuild from source currently, which causes log files that exceed the 4mb limit. The other tests passed, so I suppose everything is working.

**Comment by 7c6f434c** [comment]
Wait, wait, precompiled shared objects in the source tree do not sound like a source build… Is there a way to actually build it from actual source?

**Comment by namore** [comment]
@7c6f434c, it is possible. They just bundle dependencies at specific versions as binaries for convenience. Also, the documentation points out that while it is possible to use a system-wide installed version of these dependencies, "the exact details of how to do so are beyond the scope of [the documentation] and are left up to those with sufficient knowledge of the Tcl package system."
Even more warning: "installing all packages globally is untested, and as such your
mileage may vary if you do attempt to do such an install."
In summary, I suppose it's easiest/safest to stick with the gorilla-bin package for now.
